src/app/item-review/item-review.component.js

```diff
@@ -92,6 +97,20 @@ function ItemReviewController(dimDestinyTrackerService, $scope, $rootScope) {
 
   vm.reviewLabels = [5, 4, 3, 2, 1];
 
+  vm.defs = {};
+
+  getDefinitions().then((defs) => {
+    vm.reviewModeOptions = [
+      { key: 0, value: $i18next.t('DtrReview.ModeNotSpecified') },
+      { key: 7, value: defs.ActivityMode[1164760493].displayProperties.name },
+      { key: 5, value: defs.ActivityMode[1164760504].displayProperties.name },
+      { key: 4, value: defs.ActivityMode[2043403989].displayProperties.name },
+      { key: 39, value: defs.ActivityMode[1370326378].displayProperties.name }
+    ];
```

Comment:
Note: you could also turn `getReviewMode` into an Angular filter rather than repeating these here